Tests/MongoSwiftTests/ReadWriteConcernTests.swift

```diff
@@ -30,19 +30,19 @@ final class ReadWriteConcernTests: MongoSwiftTestCase {
     func testReadConcernType() throws {
```

Comment:
It's probably worth duplicating all these tests to use the string initializer too (and to cover the `custom` case)